app/utils/shapes/index.ts

```diff
@@ -0,0 +1,144 @@
+import type TribontShape from 'mow-registry/models/tribont-shape';
+import Octagon from './octagon';
+import Dimension from 'mow-registry/models/dimension';
+import Rectangle from './rectangle';
+import type Unit from 'mow-registry/models/unit';
+import type Store from '@ember-data/store';
+import SignpostWithPoint from './signpostWithPoint';
+import InvertedTriangle from './invertedTriangle';
+import Triangle from './triangle';
+import Circular from './circular';
+import Diamond from './diamond';
+import type QuantityKind from 'mow-registry/models/quantity-kind';
+import type TribontShapeClassificationCode from 'mow-registry/models/tribont-shape-classification-code';
+
+export const DIMENSIONS = {
```

Comment:
hardcoded IDs are a bit worrying, I'd much rather see the URIs here, and then dynamically fetch the ids